mypy.ini

```
@@ -2,4 +2,4 @@
 ignore_missing_imports = True
 install_types = True
 non_interactive = True
-exclude = (data_structures/stacks/next_greater_element\.py|graphs/boruvka\.py|graphs/breadth_first_search\.py|graphs/breadth_first_search_2\.py|graphs/check_cycle\.py|graphs/finding_bridges\.py|graphs/greedy_min_vertex_cover\.py|graphs/random_graph_generator\.py|maths/average_mode\.py|maths/gamma_recursive\.py|maths/proth_number\.py|maths/series/geometric_series\.py|maths/series/p_series\.py|matrix/matrix_operation\.py|other/fischer_yates_shuffle\.py|other/least_recently_used\.py|other/lfu_cache\.py|other/lru_cache\.py|searches/simulated_annealing\.py|searches/ternary_search\.py)/$
+exclude = /(data_structures/stacks/next_greater_element\.py|graphs/boruvka\.py|graphs/breadth_first_search\.py|graphs/breadth_first_search_2\.py|graphs/check_cycle\.py|graphs/finding_bridges\.py|graphs/greedy_min_vertex_cover\.py|graphs/random_graph_generator\.py|maths/average_mode\.py|maths/gamma_recursive\.py|maths/proth_number\.py|maths/series/geometric_series\.py|maths/series/p_series\.py|matrix/matrix_operation\.py|other/fischer_yates_shuffle\.py|other/least_recently_used\.py|other/lfu_cache\.py|other/lru_cache\.py|searches/simulated_annealing\.py|searches/ternary_search\.py)$
```

@cclauss 
This should work
```suggestion
exclude = (data_structures/stacks/next_greater_element.py|graphs/boruvka.py|graphs/breadth_first_search.py|graphs/breadth_first_search_2.py|graphs/check_cycle.py|graphs/finding_bridges.py|graphs/greedy_min_vertex_cover.py|graphs/random_graph_generator.py|maths/average_mode.py|maths/gamma_recursive.py|maths/proth_number.py|maths/series/geometric_series.py|maths/series/p_series.py|matrix_operation.py|other/fischer_yates_shuffle.py|other/least_recently_used.py|other/lfu_cache.py|other/lru_cache.py|searches/simulated_annealing.py|searches/ternary_search.py)

```